### DISCUSSION TITLE
[source-netsuite] Connector 400 error create connector

### DISCUSSION THREAD

**Discussion by pallvue** [airbytehq/airbyte#34188] [Connector Questions]
I am having trouble setting up the NetSuite connector for Airbyte OSS.

I have followed the documentation found here exactly:
https://docs.airbyte.com/integrations/sources/netsuite/

I have also reviewed this very similar thread and cannot find anything useful in it besides recreating the NetSuite Token.
https://discuss.airbyte.io/t/airbyte-netsuite-connector-400-client-error/2948

I receive this error when trying to set up source
The connection tests failed.HTTPError('400 Client Error: Bad Request for url: https://{accountId}.suitetalk.api.netsuite.com/services/rest/record/v1/contact?limit=1')

Any help is appreciated.

<img width="767" alt="image" src="https://github.com/airbytehq/airbyte/assets/156369028/cc6f9aee-5cd0-4240-8710-bebd6812386e">

**Comment by marcosmarxm** [comment]
Hello @pallvue at the moment Airbyte team is working in a new version of the Netsuite connector uses ODBC backend. You can check more information going into the pull request https://github.com/airbytehq/airbyte/pull/32968. Probably the current version will be deprecated soon.

**Comment by nataliamordvanyuk-s4l** [comment]
Hi there! I am experiencing the same issue when trying to connect NetSuite using the Airbyte community connector. I get a 400 Client Error: Bad Request when testing the connection

`400 Client Error: Bad Request for url: https://6938861.suitetalk.api.netsuite.com/services/rest/record/v1/contact?limit=1'),additionalProperties={}]"`

Could you please advise on any possible fixes or workarounds? Let me know if you need additional details about my setup. Thanks!

[a comment by an automated account is not shown]
